frontend/app/src/components/settings/general/nft/NftImageRenderingSettingMenu.vue

```diff
@@ -16,7 +13,9 @@
       </RuiButton>
     </template>
     <div class="p-4">
-      <NftImageRenderingSetting />
+      <VApp>
+        <NftImageRenderingSetting />
+      </VApp>
```

Comment:
what problem do we have without v-app here?

Reply:
So I use this for menu content that still has Vuetify component.

Because the menu is teleported to the body, so it's outside the main VApp, the result is these vuetify components have broken styling.

I cannot teleport it inside current VApp instead because it's messing with the scrolling.

Reply:
OK, so temporary measure until we remove all vuetify components from the menu then. I guess it's mostly autocomplete/select?

Reply:
yes, for this case, it's autocomplete and combobox